src/compile-middleware.ts

```diff
@@ -52,7 +52,7 @@ export const babelCompile: RequestHandler = transformResponse({
         compileMimeTypes.indexOf(getContentType(response)) >= 0;
   },
 
-  transform(request: Request, response: Response, body: string): string{
+  transform(request: Request, response: Response, body: string): string /* */ {
```

Comment:
This is still going on, I just had to modify it slightly for clang to not wrap the line.